src/Model.php

```diff
@@ -625,15 +625,21 @@ public function isDirty(string $field): bool
      *
      * @return Field[]
      */
-    public function getFields($filters = null): array
+    public function getFields($filters = null, $onlyFields = null): array
```

Comment:
`bool $onlyFields`?

Reply:
Changed

Reply:
but why to null by default?

Reply:
To keep BC

`$model->getFields()` returns all fields historically
`$model->getFields('system')` returns system fields in onlyFields

So if `$filter` argument set `$onlyFields` defaults to `true`